### ISSUE TITLE
Allow wildcard in executable names

### ISSUE THREAD

**Issue by SuperSandro2000** [dim-an/cod#26]
I want to ignore all nix commands which have the style ``nix``, ``nix-build``, ``nix-shell``, etc. This is currently not possible cause wildcard can only ignore whole directories.

**Comment by dim-an** [comment]
Yeah, implementing good wildcard matching algorithm like in `zsh` looked pretty complicated, so in current version I've used simple algorithm that covers most of my needs. 

Though it would be nice feature to have.

**Comment by SuperSandro2000** [comment]
I think this is pretty easily possible with https://godoc.org/github.com/minio/minio/pkg/wildcard

**Comment by SuperSandro2000** [comment]
> ~/ -- for home directory

Replace ~ at the beginning with $HOME.

I don't use zsh and I don't think it is required for now to get its more regex like globbing.

I would simply replace the * handling code to get my problem fixed and leave the other things to you.

**Comment by SuperSandro2000** [comment]
I did a PR with minios wildcard parsing https://github.com/dim-an/cod/pull/27.

I hacked it quickly together and you probably shouldn't merge it like this but it works for my usecase.
